index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <title>Tic-Tac-Toe!</title>
+  </head>
+<body>
+  <div id="box">
+    <h1>Let's Play Tic-Tac-Toe!</h1>
+    <h2>Instructions: This is a two-player game. The first person starts by placing their "x" on the board with a mouse click.</h2>
+
+      <canvas id = "square1" onclick="canvasClicked(1)"></canvas>
+      <canvas id = "square2" onclick="canvasClicked(2)"></canvas>
+      <canvas id = "square3" onclick="canvasClicked(3)"></canvas>
+      <p></p>
+      <canvas id = "square4" onclick="canvasClicked(4)"></canvas>
+      <canvas id = "square5" onclick="canvasClicked(5)"></canvas>
+      <canvas id = "square6" onclick="canvasClicked(6)"></canvas>
+      <p></p>
+      <canvas id = "square7" onclick="canvasClicked(7)"></canvas>
+      <canvas id = "square8" onclick="canvasClicked(8)"></canvas>
+      <canvas id = "square9" onclick="canvasClicked(9)"></canvas>
```

Comment:
I don't recognize this "canvas" tag.  My google search isn't turning up anything either.  Did we cover this in class?